README.md

```diff
@@ -1,149 +1,43 @@
 
-## Import the example application
+Depending on the target, the example project can be built with `GCC_ARM`, `ARM` or `IAR` toolchain. Run the command below after installing ARM Mbed CLI to determine which toolchain supports your target.
 
-From the command-line, import the example:
-
-```
-mbed import mbed-os-example-blinky
-cd mbed-os-example-blinky
-```
-
-### Now compile
-
-Invoke `mbed compile`, and specify the name of your platform and your favorite toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the Arm Compiler:
-
-```
-mbed compile -m K64F -t ARM
+```bash
+$ mbed compile -S
 ```
 
-Your PC may take a few minutes to compile your code. At the end, you see the following result:
-
-```
-[snip]
-
-Image: ./BUILD/K64F/GCC_ARM/mbed-os-example-blinky.bin
-```
-
-### Program your board
-
-1. Connect your Mbed device to the computer over USB.
-1. Copy the binary file to the Mbed device.
-1. Press the reset button to start the program.
+Clone this repository on your system and change the current directory to where the project was cloned.
 
-The LED on your platform turns on and off. The main thread will additionally take a snapshot of the device's runtime statistics and display it over serial to your PC. The snapshot includes:
 
-* System Information:
-    * Mbed OS Version: Will currently default to 999999
-    * Compiler ID
-        * ARM = 1
-        * GCC_ARM = 2
-        * IAR = 3
-    * [CPUID Register Information](#cpuid-register-information)
-    * [Compiler Version](#compiler-version)
-* CPU Statistics
-    * Percentage of runtime that the device has spent awake versus in sleep
-* Heap Statistics
-    * Current heap size
-    * Max heap size which refers to the largest the heap has grown to
-* Thread Statistics
-    * Provides information on all running threads in the OS including
-        * Thread ID
-        * Thread Name
-        * Thread State
-        * Thread Priority
-        * Thread Stack Size
-        * Thread Stack Space
+## Application functionality
 
-#### Compiler Version
+The `main()` function is the single thread in the application, it toggles the state of a digital output connected to an LED on the target.
 
-| Compiler | Version Layout |
-| -------- | -------------- |
-| ARM      | PVVbbbb (P = Major; VV = Minor; bbbb = build number) |
-| GCC      | VVRRPP  (VV = Version; RR = Revision; PP = Patch)    |
-| IAR      | VRRRPPP (V = Version; RRR = Revision; PPP = Patch)   |
+## Building and Running
 
-#### CPUID Register Information
+1. Connect a USB cable between the USB port on the target and the host computer.
+2. <a name="build_cmd"></a> Run the following command to build the example project and program the microcontroller flash memory:
+    ```bash
+    $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash
+    ```
+The binary is located at `./BUILD/<TARGET>/<TOOLCHAIN>/mbed-os-example-blinky.bin` and can alternatively be manually copied to the target which gets mounted on the host computer via USB.
```

Comment:
As a template for other examples, add an "Output" section.
In this case, just write: `The LED on your platform turns on and off every 500ms. Nothing is printed on the serial terminal.`

Reply:
[Added](https://github.com/ARMmbed/mbed-os-example-blinky/pull/192/commits/96c4eba10ef3d667af8beeb46da52f32e3664540) with slightly modified text.